Bottom: a295c2070e66415932630202e8ccef10479ac374
Top:    f6a471a09c445f5421f56f368077ffb445c3cf0e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-23 16:12:45 +0100

Refresh of draw-c-combine-front-and-back-

---

diff --git a/src/draw.c b/src/draw.c
index 6f76cca..20b8f46 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -250,7 +250,7 @@ name_callback (const BoxType * b, void *cl)
   if (TEST_FLAG (HIDENAMEFLAG, element))
     return 0;
 
-  if (TEST_FLAG (ONSOLDERFLAG, element) == (*side == SOLDER_LAYER))
+  if (ON_SIDE (element, side))
     DrawElementName (element, 0);
   return 0;
 }
